src/lib.rs

```diff
@@ -226,46 +210,20 @@ impl Preprocessor for KatexProcessor {
     async fn run(&self, ctx: &PreprocessorContext, mut book: Book) -> Result<Book, Error> {
         // parse TOML config
         let cfg = get_config(&ctx.config)?;
-        if cfg.static_css {
-            // enforce config requirements
-            enforce_config(&ctx.config);
-        }
         let (inline_opts, display_opts, extra_opts) = cfg.build_opts(&ctx.root);
-        // get stylesheet header
-        let (stylesheet_header, maybe_download_task) =
-            katex_header(&ctx.root, &ctx.config.build.build_dir, &cfg).await?;
-        let mut paths_w_raw_contents = Vec::new();
+        let header = if cfg.no_css { "" } else { KATEX_HEADER }.to_owned();
+        let mut tasks = Vec::with_capacity(book.sections.len());
         book.for_each_mut(|item| {
             if let BookItem::Chapter(chapter) = item {
-                if let Some(ref path) = chapter.path {
```

Comment:
We don't check whether a `chapter` has `path` anymore since we don't use `path`.
I have not encountered this situation in testing.
No idea what the implication would be.